### DISCUSSION TITLE
Country names are not in English

### DISCUSSION THREAD

**Discussion by yusufsaifii** [Daniel-Ioannou/flutter_country_picker#83] [Q&A] [answered]
Thanks for the library,
I implemented this in my application it works perfectly, but name of countries are not in English, 
It shows country name by the country language,
How I can make it to display all country names in English?

**Comment by yusufsaifii** [comment]
![Screenshot_7](https://user-images.githubusercontent.com/36518052/184958941-ab66fcd2-03c3-449e-93bb-3190939fd1ea.png)

**Comment by Daniel-Ioannou** [comment, marked as answer]
@yusufsaifii I think you are using another package, the UI does not appear to be from this package. 

This package display country names in English or any of the supported locales. 
If you want to use this package please check the [documentation](https://pub.dev/packages/country_picker). If you have any questions please feel free to ask.
